### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by Wishbringer**

### CODE REVIEW

File: includes/uveto/freezer.as [hunk 1]
```diff
@@ -303,7 +303,7 @@ public function hanaTalkEyepatch():void
...
 	
-	output("\n\n<i>“This... this was </i>my<i> reward for somebody else’s mistake. From someone playing hero - and </i>failing<i>. It came when my dear husband got himself killed trying to save someone else.... I don’t want to </i>talk<i> about it, traveler.”</i> Her tone is gravelly, stern with the unmistakable harsh edge of somebody speaking through gritted teeth. A sore subject, obviously. <i>“I am haleogentic. I can’t ‘fix’ it. I don’t </i>want<i> to fix it. Don’t ask about it again.”</i>");
+	output("\n\n<i>“This... this was </i>my<i> reward for somebody else’s mistake. From someone playing hero - and</i> failing<i>. It came when my dear husband got himself killed trying to save someone else.... I don’t want to </i>talk<i> about it, traveler.”</i> Her tone is gravelly, stern with the unmistakable harsh edge of somebody speaking through gritted teeth. A sore subject, obviously. <i>“I am haleogentic. I can’t ‘fix’ it. I don’t </i>want<i> to fix it. Don’t ask about it again.”</i>");
```
Comment (Wishbringer) [inline includes/uveto/freezer.as]: Looked like the was a missing " " on my end, can anyone replicate?

File: includes/uveto/jerome.as [hunk 2]
```diff
@@ -384,7 +384,7 @@ public function jeromePeggingIIIBiped():void
 		
 		output("\n\nWith a grunt you push forth one final time, your knot briefly held back by the tight ring threatening to reject its entry before being swallowed whole with an audible ‘POP!’.");
 		
-		output("\n\nThe fat intrusion nestling tight against the his prostate finally sets the bear off, his cock cradled tight");
+		output("\n\nThe fat intrusion nestling tight against the prostate finally sets the bear off, his cock cradled tight");
```
Comment (Wishbringer) [inline includes/uveto/jerome.as]: Not sure if "the" or "his" is better here, went with "the", maybe "his" is better?

File: classes/Characters/Kara.as [hunk 3]
```diff
@@ -156,6 +156,7 @@
 			this.vaginas[0].loosenessRaw = 4;
 			this.vaginas[0].wetnessRaw = 5;
 			this.vaginas[0].bonusCapacity = 300;
+			this.vaginas[0].hymen = false;
```
Comment (Wishbringer) [inline classes/Characters/Kara.as]: Pardon, my OCD kicked in.

File: classes/Characters/Jerome.as [hunk 4]
```diff
@@ -165,7 +165,7 @@ package classes.Characters
 			//The rate at which you produce milk. Scales from 0 to INFINITY.
 			this.milkRate = 0;
 			this.ass.wetnessRaw = 0;
-			ass.loosenessRaw = 4;
+			this.ass.loosenessRaw = 4;
```
Comment (Fenoxo) [inline classes/Characters/Jerome.as]: This isn't actually needed on these.

File: includes/mhenga/roomFunctions.as [hunk 5]
```diff
@@ -42,7 +42,7 @@ public function xenogenOutsideBlurb():Boolean
 
 public function synthSapNoticeUnlock():Boolean
 {
-	return (flags["MET_VANAE_MAIDEN"] != undefined && flags["MET_VANAE_HUNTRESS"] != undefined && CodexManager.entryViewed("Vanae: History"));
+	return (flags["MET_VANAE_MAIDEN"] != undefined && flags["MET_VANAE_HUNTRESS"] != undefined && CodexManager.entryViewed("Vanae"));
```
Comment (Wishbringer) [inline includes/mhenga/roomFunctions.as]: This quest wasn't working, but was it intentional? Is the codex supposed to support subsection checks?

File: includes/uveto/jerome.as (hunk 2 above)
Comment (Fenoxo) [inline includes/uveto/jerome.as]: Definitely his.

File: includes/newTexas/treatment.as [hunk 6]
```diff
@@ -3061,8 +3061,10 @@ public function treatedVagNote(butt:Boolean = false):void
 }
 public function amazonEnduranceNotice():void
 {
-	eventBuffer += "\n\n" + logTimeStamp() + " Usually, sexual release would leave you feeling a little drained, but you're still full of energy. It's like cumming isn't slowing you down at all, like your body is designed to get off with a minimum of effort so that you can keep banging until your partner's body gives out or begins to bore you.";
-	eventBuffer += "\n\n(<b>Perk Gained: Amazonian Endurance</b> - Allows you to orgasm without fatiguing.)";
+	var eBuffer:String = "";
+	eBuffer += "\n\n" + logTimeStamp() + " Usually, sexual release would leave you feeling a little drained, but you're still full of energy. It's like cumming isn't slowing you down at all, like your body is designed to get off with a minimum of effort so that you can keep banging until your partner's body gives out or begins to bore you.";
+	eBuffer += "\n\n(<b>Perk Gained: Amazonian Endurance</b> - Allows you to orgasm without fatiguing.)";
+	eventBuffer += ParseText(eBuffer);
 }
```
Comment (Fenoxo) [inline includes/newTexas/treatment.as]: Completely pointless changes here since there is nothing in the text that needs parsed.

Comment (Wishbringer) [inline includes/newTexas/treatment.as]: Well, the timestamp looked weird.

File: classes/Characters/Jerome.as (hunk 4 above)
Comment (Wishbringer) [inline classes/Characters/Jerome.as]: Yeah, "this." is implied, but it looks better if everything uses same syntax.

File: includes/mhenga/roomFunctions.as (hunk 5 above)
Comment (Jacques00) [inline includes/mhenga/roomFunctions.as]: Please don't activate this--it was supposed to be for an expansion that has yet to be reviewed and implemented...

File: includes/uveto/jerome.as [hunk 7]
```diff
@@ -384,7 +384,7 @@ public function jeromePeggingIIIBiped():void
 		
 		output("\n\nWith a grunt you push forth one final time, your knot briefly held back by the tight ring threatening to reject its entry before being swallowed whole with an audible ‘POP!’.");
 		
-		output("\n\nThe fat intrusion nestling tight against the prostate finally sets the bear off, his cock cradled tight");
+		output("\n\nThe fat intrusion nestling tight against his prostate finally sets the bear off, his cock cradled tight");
```
Comment (Wishbringer) [inline includes/uveto/jerome.as]: "his" it is.

File: includes/mhenga/roomFunctions.as (hunk 5 above)
Comment (Wishbringer) [inline includes/mhenga/roomFunctions.as]: Ah, so it was intentional, something felt weird.

File: includes/newTexas/treatment.as (hunk 6 above)
Comment (Fenoxo) [inline includes/newTexas/treatment.as]: Ah, I guess I could see that.

File: classes/Characters/Jerome.as (hunk 4 above)
Comment (Fenoxo) [inline classes/Characters/Jerome.as]: We don't need to spend time making code changes entirely to make it prettier.

Comment (Wishbringer) [inline classes/Characters/Jerome.as]: Lets just say I have a mental problem, and can't help myself. :/

File: includes/newTexas/treatment.as [hunk 8]
```diff
@@ -3061,7 +3061,7 @@ public function treatedVagNote(butt:Boolean = false):void
...
 public function amazonEnduranceNotice():void
 {
-	eventBuffer += "\n\n" + logTimeStamp() + " Usually, sexual release would leave you feeling a little drained, but you're still full of energy. It's like cumming isn't slowing you down at all, like your body is designed to get off with a minimum of effort so that you can keep banging until your partner's body gives out or begins to bore you.";
+	eventBuffer += "\n\n" + ParseText(logTimeStamp()) + " Usually, sexual release would leave you feeling a little drained, but you're still full of energy. It's like cumming isn't slowing you down at all, like your body is designed to get off with a minimum of effort so that you can keep banging until your partner's body gives out or begins to bore you.";
```
Comment (Wishbringer) [inline includes/newTexas/treatment.as]: Looking at the logTimeStamp() function, seems like it was the "\" characters causing it.

File: classes/Characters/Jerome.as (hunk 4 above)
Comment (JustGregory) [inline classes/Characters/Jerome.as]: Mental or not, I actually prefer without the "this." prefix. Is there really any place where having it versus not having it makes a functional difference?

Comment (Wishbringer) [inline classes/Characters/Jerome.as]: Same here @JustGregory, though conformity of style is more important when working with others.
"this." is only useful when there are potential conflicts.

**Pull request merged**